.github/workflows/tox.yml

```diff
@@ -16,6 +16,7 @@ jobs:
           - '3.8'
           - '3.9'
           - '3.10'
+          - '3.11'
     runs-on: ubuntu-latest
```

Comment:
Do we also want to explicit set the version of ubuntu i.e. 22.04 like in omero-py?

Reply:
See [b917663](https://github.com/ome/omero-dropbox/pull/22/commits/b9176636094841ba7c81dd7b06d9a2c46ca0ce0c)